### ISSUE TITLE
Tropical fish material sliders don't work

### ISSUE THREAD

**Issue by TheDuckCow** [TheDuckCow/MCprep#341]
### Did the issue happen after restarting Blender?

- [X] Yes, the issue indeed continued after restarting Blender

### Check known issues

- [X] I have checked known issues, and my problem is different

### What happened?

It appears that the tropical fish material was completely reset somewhere along the process in making it accessible for MCprep users. We will need to resource from the original pack and replicate materials as necessary, and then add it to the json list of materials to not auto-prep (so by default it works the best for the majority of users).

### What steps are needed to produce the bug?

Spawn the tropical fish

### What version of MCprep are you using?

MCprep 3.4.1

### What version of Blender are you using?

All of them :)

**Comment by TheDuckCow** [comment]
I looked at this briefly, and it looks to be a more fundamental problem - the drivers have been completely lost, so the materials would need to be remade.

I propose that we treat it similar to the 2.8 villager, and should be a 2.8+ only rig. Moving it out of this next release.

**Comment by TheDuckCow** [comment]
@MajesticalDiscomfort to take a look

**Comment by StandingPadAnimations** [comment]
Seeing as we're getting close to the end of the MCprep 3.5 release cycle, I'll delay this for 3.5.1

**Comment by TheDuckCow** [comment]
Agreed on this one - I am, however, going to prioritize doing the QA for the other rigs that were already submitted. That's on me to do, following the guidelines we recently discussed (I have a branch local with those guidelines changes + some tweaked wording, not yet pushed)

**Comment by StandingPadAnimations** [comment]
Following up for some progress updates on the issue. Should we delay this further for MCprep 3.5.3 or reassign the issue to someone else?

**Comment by StandingPadAnimations** [comment]
I'm just going to reassign the issue to @zNightlord

**Comment by zNightlord** [comment]
![image](https://github.com/Moo-Ack-Productions/MCprep/assets/61040487/afe4d5c8-f52b-48c4-9b71-e7399597ff09)
[MCprep Discord Message Blend](https://discord.com/channels/737871405349339232/741468426404495410/1193478742148907058)

I guess the last problem here is how prep material kill on spawning the rig in, it can simply resolve by adding the rig into the json file data *for now*

**Comment by StandingPadAnimations** [comment]
Adding this to the MCprep 3.6.1 milestone so we can get this fixed in the near future
